client-app/pages/account/order-details.vue

```diff
@@ -6,19 +6,26 @@
 
     <div class="max-w-screen-2xl md:px-12 mx-auto">
       <h2 class="text-gray-800 px-5 md:px-0 text-2xl lg:text-3xl font-bold uppercase mb-2">
-        Order #{{ order?.number }}
+        {{ $t("pages.account.order_details.title_prefix") }}{{ order?.number }}
```

Comment:
Need to use formatted localized string. See https://vue-i18n.intlify.dev/guide/essentials/syntax.html as an example.